pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md

```diff
@@ -1,36 +1,36 @@
 ---
 layout: layout.pug
 beta: true
-navigationTitle: Federated Addons
-title: Federated Addons
+navigationTitle: Federated Platform Services
+title: Federated Platform Services
 menuWeight: 5
-excerpt: How federated addons work
+excerpt: How federated platform services work
 ---
 
-## Addons federation
+## Platform services federation
 
-When attaching a cluster, Kommander federates certain addons on the newly attached cluster. To customize the federation of the logging and monitoring stacks, operators can apply labels to the associated `KommanderCluster` resource.
+When attaching a cluster, Kommander federates certain platform services on the newly attached cluster. To customize the federation of the logging and monitoring stacks, operators can apply labels to the associated `KommanderCluster` resource.
 
 On attachment, two factors impact successfully deploying an addon on the attached cluster:
 
 1. Is the attached cluster a Konvoy cluster or not? For example, a cluster deployed using AWS EKS.
 2. Does a label describing the federation of the addon exist and is it defined? If yes, what is its value?
 
-The following tables describe the list of addons and cluster addons that get federated on attachment with their related federation label (if available). Addons that do not have a federation label are federated by default. If the addon description indicates only federated on non-Konvoy clusters, the addon will not get installed into Konvoy clusters, even if its federation label is set to `true`.
+The following tables describe the list of platform services and cluster platform services that get federated on attachment with their related federation label (if available). Platform services that do not have a federation label are federated by default. If the addon description indicates only federated on non-Konvoy clusters, the addon will not get installed into Konvoy clusters, even if its federation label is set to `true`.
```

Comment:
I don't really know, but should these other instances of platform services be included in this graf?

```suggestion
The following tables describe the list of platform services and cluster platform services that get federated on attachment with their related federation label (if available). Platform services that do not have a federation label are federated by default. If the platform service description indicates only federated on non-Konvoy clusters, the platform service will not get installed into Konvoy clusters, even if its federation label is set to `true`.
```

Reply:
oops yes I missed those